openmetadata-ui/src/main/resources/ui/src/components/Chart/DataDistributionHistogram.component.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Col, Row, Tag } from 'antd';
+import ErrorPlaceHolder from 'components/common/error-with-placeholder/ErrorPlaceHolder';
+import { GRAPH_BACKGROUND_COLOR } from 'constants/constants';
+import { DEFAULT_HISTOGRAM_DATA } from 'constants/profiler.constant';
+import { HistogramClass } from 'generated/entity/data/table';
+import { isUndefined, map } from 'lodash';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { axisTickFormatter, tooltipFormatter } from 'utils/ChartUtils';
+import { getFormattedDateFromSeconds } from 'utils/TimeUtils';
+import { DataDistributionHistogramProps } from './Chart.interface';
+
+const DataDistributionHistogram = ({
+  data,
+}: DataDistributionHistogramProps) => {
+  const { t } = useTranslation();
+  const showSingleGraph =
+    isUndefined(data.firstDayData?.histogram) ||
+    isUndefined(data.currentDayData?.histogram);
+
+  if (
+    isUndefined(data.firstDayData?.histogram) &&
+    isUndefined(data.currentDayData?.histogram)
+  ) {
+    return (
+      <Row align="middle" className="h-full w-full" justify="center">
+        <Col>
+          <ErrorPlaceHolder>
+            <p>{t('message.no-data-available')}</p>
```

Comment:
I would recommend using the `Typography.Paragraph` here.

Reply:
thanks, will take care of this in my next PR